openedx/core/djangoapps/content/search/documents.py

```diff
@@ -546,3 +547,57 @@ def searchable_doc_for_collection(
             doc['_disabled'] = True
 
     return doc
+
+
+def searchable_doc_for_container(
+    container_key: LibraryContainerLocator,
+) -> dict:
+    """
+    Generate a dictionary document suitable for ingestion into a search engine
+    like Meilisearch or Elasticsearch, so that the given collection can be
+    found using faceted search.
+
+    If no container is found for the given container key, the returned document
+    will contain only basic information derived from the container key, and no
+    Fields.type value will be included in the returned dict.
+    """
+    doc = {
+        Fields.id: meili_id_from_opaque_key(container_key),
+        Fields.context_key: str(container_key.library_key),
+        Fields.org: str(container_key.library_key.org),
```

Comment:
Nit, the `LibraryContainerLocator` has the [org property](https://github.com/openedx/opaque-keys/pull/369/files#diff-4c80f03bddd8f564f1b21a9b75defa1c134f277c4da5a4b1cb984f0c66f62280R1716)

```suggestion
        Fields.org: str(container_key.org),
```